test/test_view_ops.py

```diff
@@ -1038,6 +1038,34 @@ def test_broadcast_shapes(self, device):
                 actual = torch.broadcast_shapes(s0, s1)
                 self.assertEqual(expected, actual)
 
+    # Skip BFloat16 since numpy does not support it
+    @dtypes(*torch.testing.get_all_dtypes(include_bfloat16=False))
+    def test_broadcast_to(self, device, dtype):
+        def can_broadcast(s0, s1):
```

Comment:
It's great this this accounts for broadcast_to increasing the dimensionality of the tensor.